serverless.yml

```diff
@@ -121,12 +168,70 @@ resources: # CloudFormation template syntax from here on.
         AttributeDefinitions:
           - AttributeName: user_id
             AttributeType: S
+          - AttributeName: username
+            AttributeType: S
         KeySchema:
           - AttributeName: user_id
             KeyType: HASH
         ProvisionedThroughput:
           ReadCapacityUnits: 1
           WriteCapacityUnits: 1
+        GlobalSecondaryIndexes:
+          - IndexName: UsernameIndex
+            KeySchema:
+              - AttributeName: username
```

Comment:
Should  the user name not be a range?

Reply:
Just like in the follower table

Reply:
Resolved by @chrsBa in 6c148977687263526bfb0dfe61d2d29560bf756d